authors/apps/helpers/share_articles.py

```diff
@@ -2,7 +2,6 @@
 from django.urls import reverse
 from django.conf import settings
 
-
```

Comment:
Hey @felixkiryowa, good job with the PR,
But you have removed many spaces and this overpopulates `file changes` section
We can always raise a PR to fine-tune the codebase
Kindly revert the changes